package.json

```
@@ -1,19 +1,37 @@
 {
   "name": "open-insights",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Nit: this should be `0.1.0`
```suggestion
  "version": "0.1.0",
```